plugins/SolarSystemEditor/resources/comet_discovery.fab

```diff
@@ -89,7 +89,7 @@
    C/1583 J1|C/1582    J1|1582       |      |1582-05-12|T. Brahe
    C/1585 T1|C/1585    T1|1585       |      |1585-10-13|T. Brahe
    C/1590 E1|C/1590    E1|1590       |      |1590-03-05|T. Brahe
-   C/1593 O1|C/1593    O1|1593       |      |1593-07-29|Ch.J. Ripensis
```

Comment:
The 2-letter abbreviations used in 16th-20th centuries were a greater help to almost always have the name right. 

Ph=Philipp vs. P=Paul/Peter/Pierre
Ch=Christ(oph(er) | ian)  vs. C=Carl (But French Ch=Charles, Chr=Christophe)

I would prefer the 2-letter style.

Reply:
Done